### ISSUE TITLE
Uber account deletion appears to currently be impossible

### ISSUE THREAD

**Issue by abigailwillow** [jdm-contrib/jdm#2173]
https://github.com/jdm-contrib/jdm/blob/c58236a963e4f558d06a3a9ecc6d746cb92dde0a/_data/sites.json#L21284-L21296

My account was marked for deletion on May 4th, as confirmed by their support. However, I can still log in and my data is still present in their systems.

Contacting support has been a fruitless endeavor, as they keep reiterating that my account was marked for deletion on May 4th, and will be deleted after 30 days. For some reason, it doesn't occur to them that May 4th was 82 days ago...

So far, I've only been getting useless responses from their support. Even when threatened with legal action under GDPR, considering I'm an EU citizen, they would not budge at all. 

Will keep this updated, but currently it appears that the system is not functional, and their support is completely clueless regarding such issues. Perhaps it should be marked impossible currently.

**Comment by tupaschoal** [comment]
Thanks for the information, I have moved it to impossible, please let us know if you eventually have success in getting the account deleted.

**Comment by abigailwillow** [comment]
I reached out to my local Data Protection Authority, and they found that Uber was at fault and negotiated for me, the account finally got deleted recently under threat of the DPA.

As a test, I tried making a new account under a throwaway email address, I will be awaiting the 30-day period to see if this account will actually get deleted or not.

So, it's not impossible, but it is incredibly difficult if their 30-day deletion period is not working as intended, and you are essentially required to threaten with GDPR law.
